### PR TITLE
Multi route stack

### PR DESCRIPTION
**Pull request by sjsadowski**
Add this fix for 19.12LTS to fix broken release 19.12.2 (Fix for #1773 )

@huge-success/sanic-core-devs  - can someone do a review and a sanity check after the automatic checks complete?

### CODE REVIEW

File: sanic/response.py [hunk 1]
```diff
@@ -118,12 +118,7 @@ def get_headers(
         )
 
         headers = self._parse_headers()
-
-        if self.status == 200:
-            status = b"OK"
-        else:
-            status = STATUS_CODES.get(self.status)
-
+        status = STATUS_CODES.get(self.status, b"UNKNOWN RESPONSE")
```
Comment (ahopkins) [inline sanic/response.py]: I thought this was added already 🤔

Comment (sjsadowski) [reply sanic/response.py]: @ahopkins yeah, but it was merged after 19.12.2 went out. It merges the fix into the branch that I cut from 19.12.2

Comment (ahopkins) [reply sanic/response.py]: got it 👍

**Comment by sjsadowski** [comment]
@ahopkins can you add the @huge-success/sanic-release-managers to the people allowed to push to protected branches? I'm finishing this up and getting complaints from GH about pushing to 19.12LTS

**Comment by ahopkins** [comment]
You should already have this. Try again one more time and let me know if it still complains.

**Comment by sjsadowski** [comment]
@ahopkins 
remote: Resolving deltas: 100% (15/15), completed with 6 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/19.12LTS.
remote: error: Required status check "continuous-integration/travis-ci" is expected. At least 1 approving review is required by reviewers with write access.
To github.com:huge-success/sanic.git
 ! [remote rejected] 19.12LTS -> 19.12LTS (protected branch hook declined)
error: failed to push some refs to 'git@github.com:huge-success/sanic.git'

**Comment by ashleysommer** [comment]
This looks to be _all_ of the commits between 19.12LTS release, up-to and including the Multi-Route-stack fix. I'm not sure if its appropriate to include those extra commits.
I created a PR today here: https://github.com/huge-success/sanic/pull/1849
That is a request to merge into 19.12LTS and includes just three bugfix commits (using the git cherrypicking feature). And includes the multi-route stack fix.

**Comment by ashleysommer** [comment]
I don't know how this got automatically closed when I deleted a branch from my private fork. This is not even my PR.
Anyway, I reopened it now.

**Comment by ahopkins** [comment]
Nice. I'll take a look tonight. 💪

**Comment by ashleysommer** [comment]
Need to make a decision between https://github.com/huge-success/sanic/pull/1849 and https://github.com/huge-success/sanic/pull/1779

I'm concerned this commit was built incorrectly, looks like it was just a merge of `master` in May back into the 19.12 branch. I don't believe that is the correct way to backport commits to an LTS branch.

**Comment by ahopkins** [comment]
Closing in favor of #1849

**Pull request closed without merging**